### PR TITLE
Stage 1: Create data model

### PR DESCRIPTION
**Pull request by Oklukeok**
**_Features:_**
- Created a new Rails App.
- Postgres database set up.
- Created tables Users, Comments, Posts, Likes

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Lucas, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe for this project adhere(s) to the following standard(s):
- [ ] All main headers from the [ReadMe template](https://github.com/microverseinc/readme-template) should be present in your ReadMe, only the FAQ section is optional.
- [ ]  The _Future Features_ section of your ReadMe includes 1-3 potential features to be added later. 
- [ ] Kindly ensure to remove the Microverse template logo. Optionally, you can replace it with your own logo if you have one.
- [ ] Kindly ensure that provided text instructions are removed and replaced with appropriate contents applicable under each section of your ReadMe.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by DammyShittu** [comment]
Hi @Oklukeok,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### You probably forgot to push your changes. Kindly implement the changes requested by the previous reviewer.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Oklukeok** [comment]
HELLO THERE!, IF THERE IS A NEW ISSUE WITH THE README FILE, PLEASE DON'T JUST COPY/PASTE A GENERIC INVALID TEST. PLEASE TELL ME WHERE IS THE MISTAKE.
THANKS

**Comment by Oklukeok** [comment]
PD: THIS IS THE SECOND TIME THAT I CHANGED IT

File: README.md [hunk 1]
```diff
@@ -127,156 +120,89 @@ To get a local copy up and running, follow these steps.
...
-
-- **[Question_2]**
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
```
Comment (rustam-tolipov) [inline README.md]: - [ ] Please ensure that you add the MIT license link to the README of your project. Additionally, kindly take the time to personalize and remove any extra lines as outlined below. 📝🔍🛠

Screenshot 2023-01-27 at 03 11 29

Comment (rustam-tolipov) [inline README.md]: [OPTIONAL] Kindly remove this [line](https://github.com/Oklukeok/blogApp/tree/dataModel#:~:text=NOTE%3A%20we%20recommend%20using%20the%20MIT%20license%20%2D%20you%20can%20set%20it%20up%20quickly%20by%20using%20templates%20available%20on%20GitHub.%20You%20can%20also%20use%20any%20other%20license%20if%20you%20wish.) from your code:

```
NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.
```

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @Oklukeok,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Oklukeok) [reply README.md]: @Rustamxon7 Hello! thanks for the hints!

**Review by mwanawabangona — APPROVED** [review]
Hi @Oklukeok 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**